src/main/java/me/xmrvizzy/skyblocker/config/SkyblockerConfig.java

```diff
@@ -67,7 +67,7 @@ public static class QuickNav {
 
         @ConfigEntry.Category("button3")
         @ConfigEntry.Gui.CollapsibleObject()
-        public QuickNavItem button3 = new QuickNavItem(true, new ItemData("bone"), "\\Pets \\(\\d+/\\d+\\)", "/pets");
+        public QuickNavItem button3 = new QuickNavItem(true, new ItemData("bone"), "Pets \\(\\d+/\\d+\\)", "/pets");
```

Comment:
still doenst work
use this `Pets( \(\d+\/\d+\))?`

Reply:
> still doenst work
> use this `Pets( \(\d+\/\d+\))?`

It did work for me, don't forget to reset that field in the config
You need to have 2 backslashes where i did put 2, because one is to have the regex escaping of the special characters and one is for the Java string escaping of the \
The string `"Pets \\(\\d+\/\\d+\\)"` will become the regex `Pets \(\d+/\d+\)`

Reply:
OK i found out, what you mean
Mine does work, it doesn't crash anymore
But ehy have changed it so that now when you have only one page, it doesn't show the page counter anymore